Formula/fontforge.rb

```diff
@@ -43,6 +43,14 @@ def install
 
       # The "extras" built above don't get installed by default.
       bin.install Dir["bin/*"].select { |f| File.executable? f }
+
+      on_macos do
+        # Fix @rpath issue on arm64
+        system "install_name_tool", "-change",
```

Comment:
Does using `CMAKE_INSTALL_RPATH` or `CMAKE_INSTALL_NAME_DIR` not work? If not, then this is probably a bug that needs to be reported upstream.